test/lz4frame.jl

```diff
@@ -98,7 +99,7 @@
             dctx = Ref{Ptr{CodecLz4.LZ4F_dctx}}(C_NULL)
             srcsize = Ref{Csize_t}(origsize)
             dstsize =  Ref{Csize_t}(8*1280)
-            decbuffer = Vector{UInt8}(1280)
+            decbuffer = @compat Vector{UInt8}(uninitialized, 1280)
```

Comment:
Why do you used `@compat` here but not in the package code which uses `uninitialized`?

Reply:
Oddly the ones in the package code didn't seem to be causing any errors when run with 0.6

Reply:
I removed the `@compat` locally and it worked. Can you try removing it and post any errors you get?

Reply:
I remember getting an error earlier, but it seems to have disappeared. I guess I'm just losing my mind.